YMChat/src/main/java/com/yellowmessenger/ymchat/models/JavaScriptInterface.java

```diff
@@ -40,12 +39,6 @@ public void receiveMessage(String s) {
             incomingEvent = new YMBotEventResponse(null, null, false);
         }
 
-        if (incomingEvent.getCode() != null && incomingEvent.getCode().equals("start-mic")) {
```

Comment:
Please check that it doesn't affect any customer

Reply:
No impact as the event was not handled